templates/coolhole-coolpoints.pug

```diff
@@ -140,7 +135,7 @@ mixin under-construction
 
 mixin legal-nonsense
   span(style="color: gray; font-size: 12px") Terms of Service:
-  div(style="overflow-y: scroll; height: 75px; font-size: 12px; color: gray")
+  div(style="overflow-y: scroll; height: 75px; font-size: 12px; color: gray; background-color: dark-gray;")
```

Comment:
nit: we could make a style for this. i didn't realize what a mess this inline style was becoming

Reply:
Agreed, we can clinch this in while making the modal CSS more broad.